test/common/grpc/async_client_impl_test.cc

```diff
@@ -0,0 +1,381 @@
+#include "common/grpc/async_client_impl.h"
+
+#include "test/mocks/buffer/mocks.h"
+#include "test/mocks/grpc/mocks.h"
+#include "test/mocks/http/mocks.h"
+#include "test/mocks/upstream/mocks.h"
+#include "test/proto/helloworld.pb.h"
+#include "test/test_common/utility.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+using testing::_;
```

Comment:
nit: in the rest of the code we have been putting the using statements outside of the Envoy namespace.